lstchain/reco/utils.py

```diff
@@ -280,6 +280,33 @@ def sky_to_camera(alt, az, focal, pointing_alt, pointing_az):
 
     return camera_pos
 
+def radec_to_camera(source_ra, source_dec, obstime, pointing_alt, pointing_az, focal):
+    """
+    Coordinate transform from source coordinate (ra, dec) in degree to camera coordinates (x, y) in distance
```

Comment:
maybe here change here to transform from sky coordinate (ra, dec) in degree to camera coordinates (x, y) in distance